og_ui/src/BundleFormAlter.php

```diff
@@ -123,12 +123,12 @@ protected function addGroupContent(array &$form, $form_state) {
     $target_types = [];
     $bundle_options = [];
     $all_group_bundles = Og::groupManager()->getAllGroupBundles();
-    foreach ($all_group_bundles as $group_entity_type => $bundles) {
+    foreach ($all_group_bundles as $entity_type => $bundles) {
```

Comment:
Nah, we are fine now that there is only one entity type / id ,the other one got hardwired to user and renamed to uid. Previously there were two pairs and it was awfully confusing.